scripts/pipeline-DEPLOY.sh

```diff
@@ -358,18 +358,24 @@ kubectl create secret generic secure-file-storage-credentials \
   --namespace "$TARGET_NAMESPACE" || exit 1
 
 #
-# Create a secret to access the registry
+# Create a policy, then a secret to access the registry
 #
-if kubectl get secret secure-file-storage-docker-registry --namespace $TARGET_NAMESPACE; then
-  echo "Docker Registry secret already exists"
+if ibmcloud iam service-policies $SERVICE_ID | grep -B 3 container-registry | grep Reader; then
```

Comment:
@data-henrik 

could you use the JSON output instead? Like there https://github.com/IBM-Cloud/secure-file-storage/blob/master/scripts/pipeline-DEPLOY.sh#L102, should be more robust than parsing the output

Reply:
I will change that and for COS. That was the place I copied it from :)

https://github.com/IBM-Cloud/secure-file-storage/blob/master/scripts/pipeline-DEPLOY.sh#L248

Reply:
@l2fprod the corrected jq query for KP and new checks for COS and CR are in